mlflow/server/js/src/components/error-boundaries/AppErrorBoundary.js

```diff
@@ -19,7 +19,7 @@ class AppErrorBoundary extends Component {
         <div>
           <h1 className={"center"}>Oops! Something went wrong.</h1>
           <h4 className={"center"}>If this error persists, please report an issue to our Github.</h4>
-          <img className="niagara" alt="Niagara falls picture." src={niagara}/>
+          <img className="niagara" alt="The niagara falls." src={niagara}/>
```

Comment:
Not sure this change is needed?

Reply:
"Line 22:  Redundant alt attribute. Screen-readers already announce `img` tags as an image. You don't need to use the words `image`, `photo,` or `picture` (or any specified custom words) in the alt prop  jsx-a11y/img-redundant-alt"
It's a strange warning, but a very simple fix.